layouts/shortcodes/pipelines-compatibility.html

```diff
@@ -1,6 +1,6 @@
 <p><i>Due to
-<a href="https://github.com/kubeflow/pipelines/issues/1700">kubeflow/pipelines#1700</a>
-and
-<a href="https://github.com/kubeflow/pipelines/issues/337">kubeflow/pipelines#337</a>,
```

Comment:
The KFP SDK currently prepares the credentials for GCP only. So, yes, the container builder only supports GCR. However, it does not mean the containers have to be in GCR once built. As long as the credentials are set up correctly to fetch the image, other registries also work.

Reply:
Thanks @gaoning777 I've adjusted the wording. PTAL.